src/_includes/partials/components/submission.njk

```diff
@@ -10,7 +10,10 @@
     "
     data-filter-language="
     {%- if submission.data.youtube.interpretation -%}
-        {{ 'asl' if submission.data.youtube.lang === 'en' else 'lsq' }},
+        {{ 'asl' if submission.data.youtube.lang === 'en' and submission.data.youtube.signLanguageUrl }},
+    {%- endif -%}
+    {%- if submission.data.youtube.interpretation -%}
+        {{ 'lsq' if submission.data.youtube.lang === 'fr' and submission.data.youtube.signLanguageUrl }},
     {%- endif -%}
```

Comment:
You probably can consolidate this a little bit as there are two `if` clauses that do the same thing (check if there's an interpretation).